packages/fx-core/resource/package.nls.json

```diff
@@ -489,6 +489,9 @@
   "core.confirmManifestQuestion.placeholder": "Confirm you've selected the correct manifest file",
   "core.aadAppQuestion.label": "Microsoft Entra app",
   "core.aadAppQuestion.description": "Your Microsoft Entra app for Single Sign On",
+  "core.convertAadToNewSchema.continue": "Continue",
+  "core.convertAadToNewSchema.warning": "Convert Microsoft Entra app manifest to new schema will overwrite the original file. Are you sure to continue?",
+  "core.convertAadToNewSchema.success": "Microsoft Entra app manifest converted to new schema successfully.",
```

Comment:
Proposed: Microsoft Entra app manifest file successfully converted to new schema.